frontend/src/modules/layout/pages/error-page.vue

```diff
@@ -36,28 +36,20 @@
       class="absolute right-0 bottom-0 mb-16 flex items-center"
     >
       <a
-        href="https://crowd.dev/discord"
+        href="https://jira.linuxfoundation.org/plugins/servlet/desk/portal/4?requestGroup=54"
         class="btn btn-link btn-link--md btn-link--primary flex items-center mr-6"
         target="_blank"
         rel="noopener noreferrer"
       >
-        <img
-          src="https://cdn-icons-png.flaticon.com/512/5968/5968756.png"
-          alt=""
-          class="mr-2 h-4 w-4 block"
-        />
-        <span class="leading-4"> Discord community</span>
+        <span class="leading-4"> Open an issue</span>
       </a>
       <a
-        href="mailto:help@crowd.dev"
+        href="https://app.slack.com/client/T02H1G4T9/C0DMD0214"
```

Comment:
Can we please change Slack link to https://linuxfoundation.slack.com/archives/C0343H278F5
Thanks!